### PR TITLE
Upgrade Guide proofing integration

### PR DESCRIPTION
**Pull request by dariavladykina**
Integrated proofing corrections for the Upgrade Guide from VT.

### CODE REVIEW

File: xml/sle_update_background.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
   <abstract>
    <para>
     This chapter provides background information on terminology, &suse; product
-    lifecycles and Service Pack releases, and recommended upgrade policies.
+    lifecycles fand Service Pack releases, and recommended upgrade policies.
```
Comment (taroth21) [inline xml/sle_update_background.xml:16]: Hi Daria, this one looks fishy to me - maybe you accidentally pressed a key (or wanted to correct something else)...

**Review by taroth21 — CHANGES_REQUESTED** [review]
Hi Daria, one change looks fishy to me as it introduces a mistake, please have a look - the rest looks good to me, thanks!

File: xml/sle_update_background.xml [hunk 2]
```diff
@@ -945,7 +945,7 @@ DAYS=14</screen>
   <title>Enabling LTSS support</title>
   <para>
    <literal>Long Term Service Pack Support</literal> (LTSS) extends the
-   lifecycle of &productname;. It is available as an extension. For more
+   lifecycle of a &productname;. It is available as an extension. For more
```
Comment (cwickert) [inline xml/sle_update_background.xml:948]: Definitely not! &productname; is either "SUSE Linux Enterprise Server" or "SUSE Linux Enterprise Desktop", and you would not use an article before the productname.

```suggestion
   lifecycle of &productname;. It is available as an extension. For more
```

File: xml/sle_update_finish.xml [hunk 3]
```diff
@@ -63,7 +63,7 @@
   </para>
   <para>
    If you find any of these files, examine their content and merge desirable
-   changes. You do not need to search the whole file system but only the
+   changes. You do not need to search the whole file system, only the
```
Comment (cwickert) [inline xml/sle_update_finish.xml:66]: What was wrong with "but"? I don't see how a colon is any better.

File: xml/sle_update_background.xml (hunk 1 above)
Comment (dariavladykina) [reply xml/sle_update_background.xml:16]: Sorry! That is very strange though - I can see in the file itself that I last touched it 12 months ago...

Comment (dariavladykina) [inline xml/sle_update_background.xml:16]:
```suggestion
    lifecycles and Service Pack releases, and recommended upgrade policies.
```

**Review by cwickert — CHANGES_REQUESTED** [review]
Too many commas for me, but may that's just me. ;-)

But the "a &productname;" definitely needs to be fixed.

**Review by cwickert — APPROVED** [review]
LGTM now.

**Pull request merged**